nilearn/image/image.py

```diff
@@ -1394,12 +1394,16 @@ def binarize_img(
         Mask image applied to mask the input data.
         If None, no masking will be applied.
 
-    two_sided : :obj:`bool`, default=True
+    two_sided : :obj:`bool`, default=False
```

Comment:
It is using `threshold_img`. It might have similar docstring for `threshold` and `two_sided` parameters as in `threshold_img`.

Reply:
yeah there is potential refactoring to do on some doc strings but better to keep for a separate PR